docs/resources/elasticsearch_logstash_pipeline.md

```diff
@@ -50,7 +50,7 @@ EOF
 }
 
 output "pipeline" {
-  value = elasticstack_elasticsearch_logstash_pipeline.example.pipeline_id
```

Comment:
This is just referencing the `pipeline_id` of the resource declared [above](https://github.com/elastic/terraform-provider-elasticstack/pull/1335/files#diff-e94732589b24df5194967b2d36273df397594453b57cb010ef93858b0e31b7f8L22) which does comply with the validation rules.